### PR TITLE
Debug CI failures

### PR DESCRIPTION
**Pull request by bricelam**
cc @nkolev92

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -85,14 +85,26 @@ jobs:
             - bash: brew install libspatialite
               displayName: Install SpatiaLite
               continueOnError: true
-            - script: eng/common/cibuild.sh --configuration $(_BuildConfig) --prepareMachine
+            - script: eng/common/cibuild.sh --configuration $(_BuildConfig) --prepareMachine --verbosity detailed
```
Comment (dougbu) [inline azure-pipelines.yml]: We normally use binary logs and stick them into the artifacts/logs.zip file to keep them around. Why the different approach here?

Comment (bricelam) [reply azure-pipelines.yml]: Just doing what the NuGet team asked me to do to help them debug.

Comment (dougbu) [reply azure-pipelines.yml]: :ok: that makes sense

**Comment by bricelam** [comment]
lol, now I can't get it to fail. Maybe updating to the latest SDK (in #14984) fixed it. It hasn't failed since.

**Comment by nkolev92** [comment]
@bricelam From the issue thread, I was under the impression that the failures (or a version of, hard to confirm all have the same root cause), have been occurring for at least ~8-9 year. 

I'd imagine we are just lucky(unlucky).

**Comment by bricelam** [comment]
We've been failing about every five builds on macOS for the past few weeks. But none have failed with the error since I updated the SDK.

**Comment by bricelam** [comment]
Could have been an unrelated issue that has been fixed.

**Comment by bricelam** [comment]
I don't think it's going to fail. Closing PR. I'll keep an eye on the builds...

**Comment by bricelam** [comment]
Still failing. Going to push https://github.com/aspnet/EntityFrameworkCore/pull/15062/commits/4e5f7aa1d80305c605dc5ac3782e5f001ba874ee to ignore failures and collect binlogs.

**Pull request closed without merging**